integration/querier_remote_read_test.go

```diff
@@ -33,6 +33,10 @@ func TestQuerierRemoteRead(t *testing.T) {
 	flags := mergeFlags(
 		BlocksStorageFlags(),
 		BlocksStorageS3Flags(),
+		map[string]string{
+			// This test writes samples sparse in time. We don't want compaction to trigger while testing.
+			"-blocks-storage.tsdb.block-ranges-period": "2h",
```

Comment:
Note to reviewers: the default in integration tests is 1m, but I don't want such a small block period range in these tests. Reason is that chunks are also cut to period ranges, making the assertion even more complicated.